packages/web-components/src/components/va-privacy-agreement/va-privacy-agreement.scss

```diff
@@ -1,9 +1,18 @@
 @import '../../mixins/accessibility.css';
+@import '../../global/variables';
 
 :host {
   display: block;
 }
 
+va-checkbox::part(checkbox) {
+  max-width: 100%;
+}
+
+.privacy-policy {
+  font-size: 1.06 * $v3-font-base-size;
```

Comment:
Can you add a comment here like "Source Sans Pro normalization for v3 - 16.96px" or something?

Reply:
This class needs to be made specific to v3 now that the if/else has been removed (`:host[uswds]`). It's affecting both v1 and v3 at the moment. And I also would like to see the class name changed to `.description`.